packages/rn-tester/js/examples/Image/ImageExample.js

```diff
@@ -1535,6 +1535,14 @@ exports.examples = [
       return <Image accessible source={fullImage} style={styles.base} />;
     },
   },
+  {
+    title: 'Accessibility Remove From Tree',
+    description:
+      ('If the `accessible` (boolean) prop is set to False, the image will be removed from the accessbility tree.': string),
+    render: function (): React.Node {
+      return <Image accessible={false} source={fullImage} style={styles.base} />;
+    },
+  },
```

Comment:
Similar note about tags, thought maybe we just skip this because the better fix is to update the test page above to add a toggle? I can look into that.

Reply:
Oh sorry, may have misunderstood. Suggestion is to add a toggle to the original 'Accessibility' test now and upstream to RN later?

Reply:
Added a toggle to the original test. Let me know if you meant something different.

Reply:
Looks good! Remaining nit is to wrap with macOS tags

Reply:
Ah, right. Thanks for the reminder!